numpy/core/_ufunc_config.py

```diff
@@ -421,8 +417,8 @@ class errstate(contextlib.ContextDecorator):
 
     Outside the context the error handling behavior has not changed:
 
-    >>> OrderedDict(sorted(np.geterr().items()))
-    OrderedDict([('divide', 'ignore'), ('invalid', 'ignore'), ('over', 'ignore'), ('under', 'ignore')])
+    >>> np.geterr()
+    {'divide': 'ignore', 'over': 'ignore', 'under': 'ignore', 'invalid': 'ignore'}
```

Comment:
How do we tell the lint check that 82 columns is OK?

Reply:
This line Matti: https://github.com/numpy/numpy/blob/3ad51382fd0e7da68daba7b7f831afc018260c37/tools/lint_diff.ini#L2

Reply:
I took this from SciPy, shall we increase/modify/remove it? To be honest, I am facing this in my PR also :)

Reply:
Is there a way to get the linter to ignore docstring output examples?

The numpydoc [style guide](https://numpydoc.readthedocs.io/en/latest/format.html), which we adhere to, still mentions 80 characters. How does scipy deal with exceptions, especially as now the CI will not run the rest of the jobs?

Reply:
I found this, which we can use:
```
--max-doc-length=n   set maximum allowed doc line length and perform these
                       checks (unchecked if not set)
```
Ref: https://pycodestyle.pycqa.org/en/latest/intro.html#example-usage-and-output

But yeah, good point on how SciPy ignores this (?)

Reply:
In some cases in SciPy, I have just added a break in the output line, and Sphinx did not complain.

Reply:
I've added `# noqa` (e.g. [what pycodestyle.py does](https://github.com/PyCQA/pycodestyle/blob/c47f5c1f977c20f16c19bc282834c70c9b40d800/pycodestyle.py#L386)) to ignore the docstring.

Reply:
Please no `#noqa`, that's just visual noise. The right thing to do here is either:
- break the line, or
- just ignore CI (it will not fail once the PR is merged, the CI job just runs on the diff in the PR)

Reply:
> Is there a way to get the linter to ignore docstring output examples?

SciPy does not do this, and it's not necessary.

Reply:
Oh wait, that's the difference; the other CI jobs in SciPy are not gated on linting passing.

Reply:
That's fairly annoying. In that case yes, we should increase the line length to 100 characters in `tools/lint_diff.ini`

Reply:
... or let the other CI jobs run even if linting is failing

Reply:
> Please no `#noqa`, that's just visual noise.

Restored, for now. I'm unsure of the best option, but am following the discussion...